BPOR/BPOR.Rms/Views/Volunteer/Consent.cshtml

```diff
@@ -0,0 +1,75 @@
+            <li>Phone number</li>
+            <li>Sex when you were born and gender identity</li>
+            <li>Ethnic group and background (for example Asian or Asian British)</li>
+            <li>Any long term conditions you have and how much it impacts your day to day life</li>
+        </ul>
+
+        <h3 class="govuk-heading-m">We use this information mainly to match you to research studies and contact you</h3>
+
+        <p class="govuk-body">We also use your information to improve our service or ask for your feedback.</p>
+
+        <h3 class="govuk-heading-m">We share this information within our team, with research study teams, but never with other parties</h3>
+
+        <p class="govuk-body">Your information is stored digitally in our secure system and we’ll never share it with other organisations or services outside of our organisation.</p>
+
+        <h3 class="govuk-heading-m">You can change your mind at any time</h3>
+
+        <p class="govuk-body">You can withdraw your consent to be contacted at any time. If you choose to close your account, we will only keep anonymous data to help us improve our service.</p>
+
+        <h3 class="govuk-heading-m">You have rights over your information if there are any problems</h3>
+
+        <p class="govuk-body">Because you are a living person whose data is being processed, you have rights over your personal information. For example, you can ask to see the information, or ask for us to stop contacting you.</p>
+
+    </div>
+</div>
+
+@using (Html.BeginForm(FormMethod.Post))
+{
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-form-group @(Html.ViewData.ModelState["AgreedToContactConsent"]?.Errors.Count > 0 ? "govuk-form-group--error" : "")">
+                <fieldset class="govuk-fieldset">
+                    <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+                        <div class="govuk-checkboxes__item">
+                            <input asp-for="AgreedToContactConsent" class="govuk-checkboxes__input" id="AgreedToContactConsent" type="checkbox" />
+                            <label class="govuk-label govuk-checkboxes__label" for="AgreedToContactConsent">
+                                I confirm that I have read and understood the <a href="https://bepartofresearch.nihr.ac.uk/site-policies/privacy-policy/" target="_blank" aria-label="Be Part of Research Privacy Policy (Opens in a new tab)">Be Part of Research Privacy Policy</a>.
+                            </label>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+        </div>
+    </div>
+
+    <div class="govuk-button-group">
+         <button type="submit" class="govuk-button">
+             Yes, I consent and wish to register now
+         </button>
+         <a asp-action="Index" asp-controller="Study" class="govuk-link" data_module="govuk-button--secondary">
```

Comment:
Do we definitely want a Cancel button here?
The GDS pattern seems to be a simple back link in the top left corner.

https://design-system.service.gov.uk/patterns/check-answers/

https://github.com/PA-NIHR-CRN/dte-study-api/blob/98b34ecf70ef546011c60518355b4af0ce4eb80d/BPOR/BPOR.Rms/Views/Account/Create.cshtml#L5-L5